pkgs/by-name/in/invidious-router/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGo122Module, fetchFromGitLab }:
+let
+  pname = "invidious-router";
+  version = "1.0";
```

Comment:
At least for pname there's no reason to have it here.

Reply:
This is because I initially used pname a few times (see @AndersonTorres review). I don't believe this does any harm or makes anything harder to read though, so I'd prefer to not address this.

Reply:
I changed that